log/log.go

```diff
@@ -0,0 +1,176 @@
+package log
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"os"
+	"runtime"
+
+	"github.com/almighty/almighty-core/configuration"
+
+	"github.com/goadesign/goa/middleware"
+	"golang.org/x/net/context"
+)
+
+var (
+	logger = NewLogger()
+)
+
+func NewLogger() *log.Logger {
+	logger := log.New()
+
+	if err := configuration.Setup(""); err != nil {
```

Comment:
Don't recreate the configuration here. Take a narrow argument into the NewLogger instead?